dex-load/src/main/scala/com/wavesplatform/dex/load/TankGenerator.scala

```diff
@@ -0,0 +1,140 @@
+    print(s"Creating orders... ")
+    val orders: mutable.HashSet[Order] = mutable.HashSet()
+    val safeAmount                     = env.assetQuantity / 2 / accounts.length / 400
+
+    accounts.foreach(acc => {
+      (0 until 399).foreach(i => {
+        val amount    = Random.nextInt(safeAmount.toInt)
+        val price     = Random.nextInt(safeAmount.toInt)
+        val pair      = pairs(Random.nextInt(pairs.length))
+        val orderType = if (i < 200 || matching) Type.BUY else Type.SELL
+
+        orders += mkOrder(env, acc, orderType, amount, price, pair)
+      })
+    })
+    println("Done")
+    Random.shuffle(orders.toList)
+  }
+
+  private def mkPlaces(seedPrefix: String, env: Environment, requestsCount: Int): Unit = {
+    print("Making requests for placing...")
+    val accounts = mkAccounts(seedPrefix, env, requestsCount / 400)
+    val assets   = mkAssets(env)
+    val pairs    = mkAssetPairs(assets)
+    val orders   = mkOrders(env, accounts, pairs, false)
+
+    distributeAssets(accounts, assets, env)
+    svRequests(env, orders)
+  }
+
+  private def mkCancels(seedPrefix: String, env: Environment, requestsCount: Int): Unit = {
+    println("Making requests for cancelling...")
+  }
+
+  private def mkMatching(seedPrefix: String, env: Environment, requestsCount: Int): Unit = {
+    println("Making requests for matching...")
+    val accounts = mkAccounts(seedPrefix, env, requestsCount / 400)
+    val assets   = mkAssets(env)
+    val pairs    = mkAssetPairs(assets)
+    val orders   = mkOrders(env, accounts, pairs)
+
+    distributeAssets(accounts, assets, env)
+    svRequests(env, orders)
+  }
+
+  private def mkOrderHistory(seedPrefix: String, env: Environment, requestsCount: Int): Unit = {
+    println("Making requests for getting order history...")
+  }
+
+  private def mkAllTypes(): Unit = {
+    println("mkAllTypes")
+  }
+
+  private def svRequests(env: Environment, orders: List[Order] = List.empty): Unit = {
+    println("All data has been generated. Now it will be saved...")
+    val requestsFile = new File("results.txt")
+    val output       = new PrintWriter(requestsFile, "utf-8")
+    try {
+      orders.foreach(o => {
+        output.println(mkPost(env, o, "/matcher/orderbook", "PLACE"))
+      })
+    } finally output.close()
+    println(s"Generated orders count: ${orders.length}")
+    println(s"Results have been saved to $requestsFile")
+  }
+
+  def mkRequests(seedPrefix: String, environment: String, requestsType: Int = 1, requestsCount: Int = 20000): Unit = {
+    val env = new Environment(ConfigFactory.parseResources(environment))
+
+    requestsType match {
+      case 1 => mkPlaces(seedPrefix, env, requestsCount)
+      case 2 => mkCancels(seedPrefix, env, requestsCount)
+      case 3 => mkMatching(seedPrefix, env, requestsCount)
+      case 4 => mkOrderHistory(seedPrefix, env, requestsCount)
+      case 5 => mkAllTypes
```

Comment:
Please use the parenthesis in methods. E.g.:
```scala
def foo(): Unit = ???
foo() // Ok
foo // Not ok

def foo: String = ???
foo // Ok
foo() // won't work
```

There is a golden rule: when you have a side effect, add parenthesis. If not - don't do this :)

Reply:
ok